src/chains/filecoin/filecoin/src/things/beacon-entry.ts

```diff
@@ -13,33 +15,49 @@ interface BeaconEntryConfig {
       serializedName: "Round";
     };
     data: {
-      type: string;
+      type: Buffer;
       serializedType: string;
       serializedName: "Data";
     };
   };
 }
 
+type C = BeaconEntryConfig;
```

Comment:
> Is C used here because you didn't want to type out BeaconEntryConfig below? If so, please change back to BeaconEntryConfig for readability.

Yes, it helps reduce a bit of time but I can change it back. I actually considered it more readable with it typed out because every single class is the same way. It ended up being noise/clutter for the some 25+ types in the latest branch that follow the exact same pattern. Let me know if you still want me to change them back to using the verbose type.

Reply:
I know this sucks man but would love for you to change them all. I reviewed it with @gnidan and @davidmurdoch just to make sure I wasn't being a stickler, and they agree the `C` syntax is confusing when not templated. Thanks!

Reply:
No problem